tests/docker-compose.override.yml

```diff
@@ -1,8 +1,24 @@
 version: "3.7"
 
 services:
+  us:
+    build:
+      context: ../datalens-us
+      dockerfile: ../datalens-us/Dockerfile
```

Comment:
TODO: remove the build completely. We need it now only for debug during development.